taxonomies/food/categories.txt

```diff
@@ -89751,12 +89751,16 @@ fr: Charcuteries de bœuf
 en: Prepared pork meats
 fr: Charcuteries de porc
 
-< en:Chicken and its products
 < en:Prepared meats
+en: Prepared poultry meats
+fr: Charcuterie de volaille
```

Comment:
We seem to use the plural for "Charcuteries de poulet", dinde etc.
```suggestion
fr: Charcuteries de volaille
```

Reply:
Ah yes, thanks